Plugin/src/main/java/dev/rosewood/rosestacker/listener/BlockListener.java

```diff
@@ -93,6 +93,11 @@ public void onBlockBreak(BlockBreakEvent event) {
         if (stackManager.isWorldDisabled(event.getPlayer().getWorld()))
             return;
 
+        if (!event.getPlayer().hasPermission("rosestacker.break")){
+            event.setCancelled(true);
```

Comment:
This literally makes it so you can't break any blocks, regardless if it's a stacked spawner or block.

Reply:
sorry that was a placement oversight will change now